bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/ExitInfoPluginConfiguration.kt

```diff
@@ -11,17 +11,38 @@ class ExitInfoPluginConfiguration(
      */
     var includeLogcat: Boolean = false,
 
+    /**
+     * Report [ApplicationExitInfo] ANRs that do not appear to correspond with BugSnag [Event]s
+     * as synthesized errors. These will appear on your dashboard without BugSnag data such as
+     * breadcrumbs and metadata, but will report crashes that BugSnag is otherwise unable to catch
+     * such as background ANRs.
+     */
+    var reportUnmatchedAnrs: Boolean = true,
+
+    /**
+     * Report [ApplicationExitInfo] native crashes that do not appear to correspond with BugSnag [Event]s
+     * as synthesized errors. These will appear on your dashboard without BugSnag data such as
+     * breadcrumbs and metadata, but will report crashes that BugSnag is otherwise unable to catch.
+     */
+    var reportUnmatchedNativeCrashes: Boolean = true,
+
     /**
      * Turn off event correlation based on the
      * [processStateSummary](ActivityManager.setProcessStateSummary) field. This can set to `true`
      * to stop `BugsnagExitInfoPlugin` overwriting the field if it is being used by the app.
      */
     var disableProcessStateSummaryOverride: Boolean = false
 ) {
-    constructor() : this(true, false, false)
+    constructor() : this(true, false, false, true, true)
```

Comment:
I think its worth naming these arguments to avoid confusion. The new arguments have been added in the wrong place.